jena-arq/src/main/java/org/apache/jena/sparql/expr/NodeValue.java

```diff
@@ -978,7 +967,6 @@ public boolean isDayTimeDuration()
     public boolean     getBoolean()     { raise(new ExprEvalTypeException("Not a boolean: "+this)) ; return false ; }
     public String      getString()      { raise(new ExprEvalTypeException("Not a string: "+this)) ; return null ; }
     public String      getLang()        { raise(new ExprEvalTypeException("Not a string: "+this)) ; return null ; }
-    public String      getCollation()   { raise(new ExprEvalTypeException("Not a sort key: "+this)) ; return null ; }
 
```

Comment:
Yup, that makes sense. See comment above. I'd be fine re-adding this method, and making it return a NodeValueSortKey, though wouldn't mind postponing it to try some generics-fu first :-)